test/support/flow_test_helper.rb

```diff
@@ -151,8 +151,26 @@ def outcome_body
       OutcomePresenter.new(flow.node(state.current_node_name), nil, state).body
     end
 
+    def outcome_title
+      raise "#{state.current_node_name} is not an outcome" unless current_node_type == :outcome
+
+      OutcomePresenter.new(flow.node(state.current_node_name), nil, state).title
+    end
```

Comment:
Rather than having this be very similar to `outcome_body` I think it'd make sense to have a

```
    def outcome_presenter
      raise "#{state.current_node_name} is not an outcome" unless current_node_type == :outcome

      OutcomePresenter.new(flow.node(state.current_node_name), nil, state)
    end
```

and then we can delegate outcome_body and outcome_title to use that